### ISSUE TITLE
A11y_SQL Azure Data Studio_Publish Changes_Publish_Keyboard:Tooltip is not visible to the Publish Changes and Remove button by getting keyboard focus.

### ISSUE THREAD

**Issue by yoginigorle** [microsoft/azuredatastudio#19416]
#### GitHub Tags:
#A11yMAS;#A11yTCS;#Win10;#WCAG2.1.1;#DesktopApp;#A11ySev3;#SH_SQL Azure Data Studio_May2022;#SQL Azure Data Studio;

## Environmental Details:
VS Code: 1.59.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0

## OS Details :
Microsoft Windows 10

## Repro Steps:
1.Add connection to the server with appropriate credentials.
2.Tab till Servers in left navigation Bar.
3.Tab till tables and Press enter
4.Tab till New tables and press enter
5.Tab till Publish changes.
6.Check the tooltip for publish changes button is visible or not.

## Note:
Same issue found for the Remove button.

## Actual results:
Tooltip is not visible to the Publish Changes and Remove button by getting keyboard focus.

## Expected results:
Tooltip should visible to the Publish Changes and Remove button by getting keyboard focus.

## User Impact:
keyboard users not able find the name of the buttons.

## Attachments:
https://user-images.githubusercontent.com/105736242/169040737-3d9bafc2-7226-4cfb-947f-cf608989f762.mp4
